homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -327,6 +342,17 @@ def __init__(
         )
 
 
+@dataclass(frozen=True)
+class WakeSettings:
+    """Settings for wake word detection."""
```

Comment:
Should VAD be an option or will we always VAD ?

Reply:
Always VAD for now. We can add an option for a simple timeout later.